src/components/project/moreProjects.tsx

```diff
@@ -1 +1,152 @@
+
+  @media (max-width: ${breakpoints.md}px) {
+    font-size: ${getRemSize(dimensions.headingSizes.medium.desktop)};
+    margin-bottom: 45px;
+  }
+
+  @media (max-width: ${breakpoints.sm}px) {
+    font-size: ${getRemSize(dimensions.headingSizes.large.mobile)};
+    margin-bottom: 32px;
+  }
+`;
+
+const StyledProjectCardsWrapper = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  width: 100%;
+
+  @media (max-width: ${breakpoints.md}px) {
+    flex-direction: column;
+  }
+`;
+const StyledProjectCard = styled.div<{ bgImage: string }>`
+  background-image: url(${(props) => props.bgImage});
+  background-size: cover;
+  background-position: center;
+  border-radius: 26px;
+  max-width: 680px;
+  min-width: 200px;
+  width: 100%;
+  // height: auto;
+  height: 264px;
+  // min-height:
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+
+  @media (max-width: ${breakpoints.md}px) {
+    max-width: 100%;
+    margin-right: auto;
+    margin-left: auto;
+    // margin-bottom: 11px;
+  }
+`;
+const StyledTitleWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 20px;
+  padding: 8px 8px 8px 24px;
+  background-color: rgba(255, 255, 255, 0.2);
+  backdrop-filter: blur(15px);
+  margin: 0px 27px 20px 20px;
+`;
+const StyledTitle = styled.h3`
+  margin: 0px;
+  font-weight: 400;
+  font-size: ${getRemSize(dimensions.textSizes.normal.desktop)};
+`;
+
+const StyledIconButton = styled(IconButton)``;
+
+const StyledTitleWrapperMobile = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 20px;
+  padding: 8px 8px 8px 24px;
+  background-color: rgba(255, 255, 255, 0.2);
+  backdrop-filter: blur(15px);
+  margin-bottom: 10px;
+  width: 100%;
+`;
+
+export function MoreProjects({ projects }: IProjectsData) {
+  const isDesktop = useIsDesktop();
+  return (
+    <>
+      <StyledHeading>More projects</StyledHeading>
+      {isDesktop ? (
+        <>
+          <StyledProjectCardsWrapper>
+            {projects.nodes.map((project, index) => (
+              <StyledProjectCard
+                key={index}
+                bgImage={project?.featuredImage?.node.sourceUrl}
+              >
+                <img
+                  src={project?.featuredImage?.node.sourceUrl}
+                  alt={`Cover Image for ${project.title}`}
+                  width={714}
+                  height={264}
+                  style={{ display: "none" }}
+                />
+                <StyledTitleWrapper>
+                  <StyledTitle>{project.title}</StyledTitle>
+                  <StyledIconButton />
+                </StyledTitleWrapper>
+              </StyledProjectCard>
+            ))}
+          </StyledProjectCardsWrapper>
+        </>
+      ) : (
+        <StyledProjectCardsWrapper>
+          {projects.nodes.map((project, index) => (
+            <>
```

Comment:
Remove Fragment

```suggestion
            
```

Reply:
By the way this Fragment introduces the warning
`Warning: Each child in a list should have a unique "key" prop.` 
Because the parent in the loop needs to have the `key={index}` :)